src/App/pages/RecipePage/RecipePage.tsx

```diff
@@ -0,0 +1,46 @@
+import { useEffect, useState } from 'react';
+
+import { WithLoader } from '@components/WithLoader';
+import { getOneRecipe } from '@services/recipes';
+import { useParams, useNavigate } from 'react-router-dom';
+
+import RecipeContent from './components/RecipeContent';
+import RecipePhoto from './components/RecipePhoto';
+import styles from './RecipePage.module.scss';
+
+export type recipe = {
+  title: string;
+  instructions: string;
+  summary: string;
+  image: string;
+  readyInMinutes: number;
+  aggregateLikes: number;
+  extendedIngredients: { original: string }[];
+};
+
+const RecipePage: React.FC = () => {
+  const id: number = Number(useParams().id);
```

Comment:
приведение к Number уже будет давать понять ts'у, какой здесь тип у переменной:
```suggestion
  const id = Number(useParams().id);
```

Reply:
Ок. Думал, что даже если он сам разбирается какой тип, то всё равно надо указать для стройности кода